xhydro/lstm_tools/lstm_functions.py

```diff
@@ -0,0 +1,949 @@
+        timeseries) variables that will be used during training.
+    x_train_static : np.ndarray
+        Tensor of size [(timesteps * watersheds) x n_static_variables] that contains the static (i.e. catchment
+        descriptors) variables that will be used during training.
+    x_train_q_stds : np.ndarray
+        Tensor of size [(timesteps * watersheds)] that contains the standard deviation of scaled streamflow values for
+        the catchment associated to the data in x_train and x_train_static. Each data point could come from any
+        catchment and this x_train_q_std variable helps scale the objective function.
+    y_train : np.ndarray
+        Tensor of size [(timesteps * watersheds)] containing the target variable for the same time point as in x_train,
+        x_train_static and x_train_q_stds. Usually the observed streamflow for the day associated to each of the
+        training points.
+    x_valid : np.ndarray
+        Tensor of size [(timesteps * watersheds) x window_size x n_dynamic_variables] that contains the dynamic (i.e.
+        timeseries) variables that will be used during validation.
+    x_valid_static : np.ndarray
+        Tensor of size [(timesteps * watersheds) x n_static_variables] that contains the static (i.e. catchment
+        descriptors) variables that will be used during validation.
+    x_valid_q_stds : np.ndarray
+        Tensor of size [(timesteps * watersheds)] that contains the standard deviation of scaled streamflow values for
+        the catchment associated to the data in x_valid and x_valid_static. Each data point could come from any
+        catchment and this x_valid_q_std variable helps scale the objective function for the validation points.
+    y_valid : np.ndarray
+        Tensor of size [(timesteps * watersheds)] containing the target variable for the same time point as in x_valid,
+        x_valid_static and x_valid_q_stds. Usually the observed streamflow for the day associated to each of the
+        validation points.
+    name_of_saved_model : str
+        Path to the model that has been pre-trained if required for simulations.
+    training_func : str
+        Name of the objective function used for training. For a regional model, it is highly recommended to use the
+        scaled nse_loss variable that uses the standard deviation of streamflow as inputs.
+    use_cpu : bool
+        Flag to force the training and simulations to be performed on the CPU rather than on the GPU(s). Must be
+        performed on a CPU that has AVX and AVX2 instruction sets, or tensorflow will fail. CPU training is very slow
+        and should only be used as a last resort (such as for CI testing and debugging).
+    seed : int, optional
+        Value to seed the random number generator to replicate tests. Set to None for operational use.
+
+    Returns
+    -------
+    code
+        Adding this just because linter will not let me put nothing.
```

Comment:
Probably because the function returns `0`, so it wants to know what that is.

Reply:
Indeed! Removed.